### ISSUE TITLE
popup into fullscreen

### ISSUE THREAD

**Issue by cavyseal** [TeamNewPipe/NewPipe#7252]
Whenever i am in popup mode and want to go back to viewing in the app it defaults into fullsrceen instead of portrait mode.  I've messed around with the settings and cant seem to get it to go into protrait when clicking back into the app. am i doing something wrong?

**Comment by Dangelman** [comment]
Switching from popup to main player should result in portrait mode, not landscape, since using popup means that you're holding the phone vertically.
I have "Start main player in fullscreen" disabled too, it's not that the problem.

**Comment by mhmdanas** [comment]
> since using popup means that you're holding the phone vertically.

@cavyseal that's not correct; you don't have to be holding the phone vertically to use the popup player.

**Comment by Dangelman** [comment]
> > since using popup means that you're holding the phone vertically.
> 
> @cavyseal that's not correct; you don't have to be holding the phone vertically to use the popup player.

You don't have to, but you usually are.

**Comment by mhmdanas** [comment]
Indeed, but that isn't enough to make us assume that it is the case and implement something that may worsen the experience for some people. I think a setting is the best here.

**Comment by Dangelman** [comment]
I've documented my everyday struggle (37s). It'd be nice if it sensed automatically your orientation instead of relying on autorotate, which I find bothersome since I like to browse while laid down.
https://streamable.com/bqa73o

**Comment by ein-bandit** [comment]
Same Issue.
Returning to app from popup forces app into Landscape orientation.
I am using fixed portrait mode too. 
Suggestion: should return to 'current system orientation' 

Graphene OS / Android 12
Pixel 4a

**Comment by kostasx31** [comment]
Same problem. Please look into it. It is not an enhancement. It is a bug.

**Comment by knuxyl** [comment]
It's been quite a while. Just sharing this is still a wanted bug fix.
